pkg/operator/osd/osd.go

```diff
@@ -103,8 +105,10 @@ func (c *Cluster) Start() error {
 			// fully resolve the storage config for this node
 			n := c.Storage.resolveNode(c.Storage.Nodes[i].Name)
 
+			resources := k8sutil.MergeResourceRequirements(c.Storage.Nodes[i].Resources, c.resources)
```

Comment:
So here, the resource requirement in Nodes, if defined, will take priority over the ones defined by the `osd` key. Right?

Reply:
Yes, the node resources take priority over cluster wide resource definitions.

Reply:
why do you do the merge only for OSDs and not for the other components?

Reply:
other components only define it at the cluster level. Only OSDs have the option to override at the node level